lib/view_component/base.rb

```diff
@@ -124,19 +126,18 @@ def render_in(view_context, &block)
       before_render
 
       if render?
-        perform_render
+        # preload content to support slot delegation and to consistently perform
+        # side-effects that may exist in the block
+        content
```

Comment:
```suggestion
```

Similar to @joelhawksley's comment (I can't see it in the files view?) this was part of a recent revert.